scripts/build-mocks.sh

```diff
@@ -2,6 +2,7 @@
 
 # generate mocks
 mockery -name ExecutionSDK -dir ./orchestrator -output ./orchestrator/mocks
+mockery -name ResultSDK -dir ./orchestrator -output ./orchestrator/mocks
```

Comment:
this mock is not used anywhere so generation is not needed.

Reply:
i updated the test and now it's used in the test